CreatureMethods.h

```diff
@@ -858,28 +858,19 @@ namespace LuaCreature
     int GetAITargets(lua_State* L, Creature* creature)
     {
 #if defined(TRINITY)
-        auto const& threatlist = creature->GetThreatManager().GetSortedThreatList();
+        auto const& threatlist = creature->GetThreatManager().GetThreatenedByMeList();
```

Comment:
Hmm. Why is trinity-specific code being changed? Probably a mistake in a merge?
Same with most changes in this file.

Reply:
It should definitely not be changed in this specific case, GetSortedThreatList is needed for AI-related methods to work on TC, GetThreatenedByMeList is wrong